pkgs/os-specific/linux/libbpf/default.nix

```diff
@@ -1,6 +1,5 @@
 { fetchFromGitHub
-, fetchpatch
-, libelf
+, elfutils
```

Comment:
I'm not sure what the policy is with backports: If you just want to unbreak 32 bit kernels this is the only line you need (-libelf +elfutils + same below in buildInputs)

That being said this version has been tested more and impact is minimal, so I'm happy either way